src/sentry/quotas/base.py

```diff
@@ -115,14 +115,11 @@ def get_organization_quota(self, organization):
 
         # If there is only a single org, this one org should
         # be allowed to consume the entire quota.
-        if settings.SENTRY_SINGLE_ORGANIZATION:
-            if system_limit < account_limit:
+        if settings.SENTRY_SINGLE_ORGANIZATION or account_limit:
+            if system_limit and (not account_limit or system_limit < account_limit / 60):
```

Comment:
Probably wouldn't be a bad idea to comment what's going on here — it's otherwise not really intuitive without coming back to this PR and finding the inline comment explaining why the right side of this expression is needed.